openapi.yml

```diff
@@ -0,0 +1,578 @@
+      responses:
+        '200':
+          description: Objects found for given druids
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ChecksumsForObjects'
+            text/csv:
+              schema:
+                type: string # AFAICT, OpenAPI does not allow us to model CSV data
+        '400':
+          description: Bad request (e.g., malformed druid(s))
+        '406':
+          description: Unsupported response format
+        '409':
+          description: Conflict - one or more missing or error-state objects
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                druids:
+                  example: ['druid:bc123df4567', 'druid:bc123df4568', 'druid:bc123df4569']
+                  type: array
+                  minItems: 1
+                  items:
+                    $ref: '#/components/schemas/Druid'
+                format:
+                  example: 'json'
+                  type: string
+                return_bare_druids:
+                  example: 'true'
+                  type: string
+                  enum:
+                    - 'false'
+                    - 'true'
+              required:
+                - druids
+                - format
+  /v1/objects/{id}/file:
+    get:
+      tags:
+        - objects
+      summary: Return a specific file from the MOAB
+      responses:
+        '200':
+          description: OK, returns a file (of whatever type, thus not validated here)
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+        '400':
+          description: Version parameter not positive integer or other problem with request params
+        '404':
+          description: Object or file or version not found
+      parameters:
+        - name: id
+          in: path
+          required: true
+          example: 'druid:bc123df4567'
+          schema:
+            $ref: '#/components/schemas/Druid'
+        - name: filepath
+          description: filepath relative to given `category` query argument, which is a directory in the MOAB
+          in: query
+          required: true
+          example: 'manifestInventory.xml'
+          schema:
+            type: string
+        - name: category
+          in: query
+          required: true
+          example: 'manifest'
+          schema:
+            type: string
+            enum:
+              - content
+              - manifest
+              - metadata
+        - name: version
+          in: query
+          required: false
+          example: '9'
+          schema:
+            type: string
+  /v1/objects/{id}/content_diff:
+    post:
+      tags:
+        - objects
+      summary: Return a content metadata diff
```

Comment:
a Moab::FileInventoryDifference for contentMetadata?   It is specific to the Moab ...

Reply:
Change made.